src/main/java/Trader.java

```diff
@@ -96,7 +96,8 @@ public static void main(String[] args) {
         List<Tradable> all_items = Arrays.asList(
                 new Horse(),
                 new Horse(),
-                new Horse()
```

Comment:
I don't know why there's a conflict here, but maybe try moving the two lines under TODO.

Reply:
I have no idea but it works on my computer